images/ose-kube-metrics-server.yml

```diff
@@ -0,0 +1,20 @@
+content:
+  source:
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      web: https://github.com/openshift/kubernetes-metrics-server
+      url: git@github.com:openshift-priv/kubernetes-metrics-server.git
+    dockerfile: Dockerfile.ocp
+enabled_repos:
+- rhel-8-baseos-rpms
+- rhel-8-appstream-rpms
+for_payload: true
+from:
+  builder:
+  - stream: golang
+  member: openshift-enterprise-base
+name: openshift/kube-metrics-server-rhel8
+payload_name: metrics-server
```

Comment:
This would be kube-metrics-server?
cc: @simonpasquier

Reply:
Yep, if that is what it gets called by other images, this is what the payload_name should be.

Reply:
IIUC `metrics-server` is what we'll use in https://github.com/openshift/cluster-monitoring-operator/blob/master/manifests/image-references to reference the kubernetes-metrics-server image. This looks ok to me.